src/nft/ZoraCreator1155Impl.sol

```diff
@@ -623,6 +698,25 @@ contract ZoraCreator1155Impl is
         }
     }
 
+    /// @notice Withdraws ETH from the Zora Rewards contract
+    function withdrawRewards(uint256 amount) public onlyAdminOrRole(CONTRACT_BASE_ID, PERMISSION_BIT_FUNDS_MANAGER) {
+        bytes memory data = abi.encodeWithSelector(IProtocolRewards.withdraw.selector, amount);
```

Comment:
what exactly is happening here? is the eth transferred first to this contract, then to the funds recipient? can you add a comment explaining this logic?